src/components/Phone.js

```diff
@@ -0,0 +1,28 @@
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import ShowIcon from '../assets/ShowIcon.png';
+
+const Phone = (props) => {
+  const {
+    name, image, brandQuery,
+  } = props;
+  return (
+    <li className="phone-container">
+    <img src={image} alt="Phone image" className='phoneimage' />
+    <h2 className="phone-name">{name}</h2>
+    <Link to={`/details/${brandQuery}`}>
+      <button type="button" className="show-icon">
+        <img src={ShowIcon} alt="Show Icon" />
```

Comment:
[REQUIRED] Whenever I click on the icon on the details page it takes me to an undefined page. I don't think this icon or the link is needed for the details page.